drivers/SmartThings/zigbee-switch/src/inovelli-vzm31-sn/init.lua

```diff
@@ -0,0 +1,445 @@
+-- Copyright 2024 Inovelli
```

Comment:
I believe this copyright will need to be changed to SmartThings.

Reply:
Thanks, I updated it.